Dockerfile

```diff
@@ -11,7 +11,6 @@ RUN apk update && apk add --no-cache make protobuf-dev=3.18.1-r1
 RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
 RUN go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
 RUN ./misc/gen_proto.sh
-
 RUN go build -buildvcs=false .
 
 FROM alpine:latest
```

Comment:
This Dockerfile is outdated/invalid. We don't generate protobuf manually anymore. Refer to the Dockerfile from the accounts service.